sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/utils.py

```diff
@@ -409,3 +418,18 @@ def raise_exception(msg, target):
             "Message needs to have multi-modal input like images.",
             ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
         )
+    if assistantMessageCount == 0:
+        raise_exception(
+            "One of the messages should have assistant role",
+            ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
+        )
+    if userMessageCount == 0:
+        raise_exception(
+            "One of the messages should have user role",
+            ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
+        )
+    if assistantMessageCount > 1:
+        raise_exception(
+            "Single Turn conversation is allowed. Only one of the messages should have user or assistant role",
```

Comment:
```suggestion
            "Only single turn conversations are allowed. Only one of the messages should have user or assistant role",
```

Reply:
+1

Reply:
Maybe: "Evaluators for multimodal conversations only support single turn. User and assistant role expected as the only role in each message."